nimare/tests/test_estimator_performance.py

```diff
@@ -404,11 +404,7 @@ def _transform_res(meta, meta_res, corr):
     #######################################
     # all combinations of meta-analysis estimators and multiple comparison correctors
     # that do not work together
-    if isinstance(meta, mkda.KDA) and corr.method == "montecarlo":
-        # TypeError: correct_fwe_montecarlo() got an unexpected keyword argument 'voxel_thresh'
-        voxel_thresh = corr.parameters.pop("voxel_thresh")
-        corr_expectation = does_not_raise()
-    elif (
+    if (
```

Comment:
@jdkent is this the correct way to update this test? Especially with regard to how `voxel_thresh` appears later on.